### DISCUSSION TITLE
Can macOS VM be integrated into VirtManager?

### DISCUSSION THREAD

**Discussion by HubKing** [notAperson535/OneClick-macOS-Simple-KVM#27] [Q&A] [answered]
I have got a macOS VM running successfully on my Manjaro Linux computer. I can run it by typing `./basic.sh`. I have been using VirtManager to run QEMU/KVM Linux desktops, and if I am not mistaken, VirtManager is some sort of GUI frontend to QEMU? Then, instead of running macOS VM  using `./basic.sh`, can I manage it within the same VirtManager?

When I had created virtual machines using Gnome Boxes, the same virtual machines automatically appeared  in VirtManager too, but this time, the macOS was not there. I wonder if it is possible.

**Comment by notAperson535** [comment, marked as answer]
@HubKing yes, look here: https://oneclick-macos-simple-kvm.notaperson535.is-a.dev/docs/manualmethod#step-2a-virtual-machine-manager
